iron-pages.html

```diff
@@ -47,9 +47,13 @@
       :host > ::content > :not(.iron-selected) {
```

Comment:
I believe that you can delete this in favor for the ::slotted one since contents are automatically switched to slots